			   READLINE PATCH REPORT
			   =====================

Readline-Release: 8.0
Patch-ID: readline80-001

Bug-Reported-by:	chet.ramey@case.edu
Bug-Reference-ID:
Bug-Reference-URL:

Bug-Description:

The history file reading code doesn't close the file descriptor open to
the history file when it encounters a zero-length file.

diff --git "a/histfile.c" "b/histfile.c"
--- a/histfile.c	2018-06-11 09:14:52.000000000 -0400
+++ b/histfile.c	2019-05-16 15:55:57.000000000 -0400
@@ -306,4 +306,5 @@
     {
       free (input);
+      close (file);
       return 0;	/* don't waste time if we don't have to */
     }
diff --git "a/patchlevel" "b/patchlevel"
--- a/patchlevel	2013-11-15 08:11:11.000000000 -0500
+++ b/patchlevel	2014-03-21 08:28:40.000000000 -0400
@@ -1,3 +1,3 @@
 # Do not edit -- exists only for use by patch
 
-0
+1
